public/app/features/admin/AdminEditUser.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { PureComponent } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import { StoreState, OrgUser } from 'app/types';
+import { loadUser, clearUser } from 'app/core/components/UserEdit/state/actions';
+import { getRouteParamsId } from 'app/core/selectors/location';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { NavModel } from '@grafana/data';
+import Page from 'app/core/components/Page/Page';
+import UserProfile from 'app/core/components/UserEdit/UserProfile';
+import UserSessions from 'app/core/components/UserEdit/UserSessions';
+import NewUserPassword from 'app/core/components/UserEdit/Admin/NewUserPassword';
+import UserPermissions from 'app/core/components/UserEdit/Admin/UserPermissions';
+import UserOrganizations from 'app/core/components/UserEdit/Admin/UserOrganizations';
+import UserStatus from 'app/core/components/UserEdit/Admin/UserStatus';
+
+export interface Props {
+  navModel: NavModel;
+  userId: number;
+  user: OrgUser;
+  loadUser: typeof loadUser;
```

Comment:
try to move these from redux to component state & actions instead, and pass the user down to components via props.